mypyc/test/test_run.py

```diff
@@ -251,3 +245,13 @@ class TestRunMultiFile(TestRun):
         'run-multimodule.test',
         'run-mypy-sim.test',
     ]
+
+
+# Run the main multi-module tests in separate compliation mode
+class TestRunSeparate(TestRun):
```

Comment:
Would it be useful to add some new test cases specific to separate compilation mode, or are the existing tests good enough?

Reply:
I'm adding two cases I remember hitting to one of the tests, but otherwise they were pretty good.

It is probably worth adding testing of manually specifying group assignments, but that will require some infrastructure so I'll leave it to a follow-up diff I think. (I've tested that manually.)